src/topics/using-containers.md

```diff
@@ -31,7 +31,7 @@ then print "Hello World" to the standard output.
 ```
 
 Before we run this, let's just break it down and see what some bits do.  Most of this
-has been explained in previous sections, the only part that is really new is the `dockerRequirement`
+has been explained in previous sections, the only part that is really new is the [`dockerRequirement`](http://w3id.org/cwl/CommandLineTool.html#DockerRequirement)
```

Comment:
Any reason for using `http` instead of `https`? Both versions work, but with `http` my browser always needs an HTTP redirect before it gets the final page (default browser security).

Reply:
I used `http` because it was what Peter provided to me in the matrix channel. But since you say both versions work, I'll change the `http` to `https`.
Thank you for pointing it out😊